management/src/main/java/org/ehcache/management/cluster/DefaultClusteringManagementService.java

```diff
@@ -82,7 +82,7 @@ public void start(ServiceProvider<Service> serviceProvider) {
     // get an ordered executor to keep ordering of management call requests
     this.managementCallExecutor = serviceProvider.getService(ExecutionService.class).getOrderedExecutor(
         configuration.getManagementCallExecutorAlias(),
-        new ArrayBlockingQueue<Runnable>(configuration.getManagementCallQueueSize()));
+        new LinkedBlockingQueue<Runnable>(configuration.getManagementCallQueueSize()));
```

Comment:
no reason... I tried this when I found a bug in ehcache close that is slow(). I'll change back.